packages/block-library/src/classic/convert-to-blocks-button.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { ToolbarButton } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { rawHandler, serialize } from '@wordpress/blocks';
+
+const ConvertToBlocksButton = ( { clientId } ) => {
+	const { replaceBlocks } = useDispatch( 'core/block-editor' );
+	const block = useSelect(
+		( select ) => {
+			return select( 'core/block-editor' ).getBlock( clientId );
+		},
+		[ clientId ]
+	);
+
+	return (
+		<ToolbarButton
+			title={ __( 'Convert to blocks' ) }
```

Comment:
Is the title required here now that we have a text label inside the button?

Reply:
Good point, removed.